src/silo/query_engine/actions/aggregated.cpp

```diff
@@ -93,26 +96,49 @@ struct Tuple {
             data_pointer += sizeof(decltype(value));
          } else if (metadata.getColumnType() == config::ColumnType::INT) {
             const int32_t value = *reinterpret_cast<const int32_t*>(data_pointer);
-            fields[metadata.name] = value;
+            if (value == INT32_MIN) {
+               fields[metadata.name] = std::nullopt;
```

Comment:
I'm not sure whether this is a good idea. What if a value at some point really is `INT32_MIN`?

Reply:
As discussed: Let's create an issue so that we can review this again at a later point:
* Either we really make int values `optional`s
* or we forbid users from inputting `INT32_MIN` in metadata and queries.